### PR TITLE
feat(editor): add module mock properties in the editor

### PR DESCRIPTION
**Pull request by MrWindlike**
### How to mock
Add `spec.exampleProperties` into the module schema, it is only used by the editor to mock the module properties.

When switching to editing the module components, the editor would set `spec.exampleProperties`'s values into `stateManager.dependencies` to inject the properties.

### Preview
image

### CODE REVIEW

**Comment by tanbowensg** [comment]
Can we enter array or object as mock data? Will they be converted correctly?

File: packages/editor/src/services/EditorStore.ts [hunk 1]
```diff
@@ -13,19 +13,20 @@ import {
...
   kind: 'app' | 'module';
   version: string;
   name: string;
+  spec?: {
+    exampleProperties?: Record<string, any>;
+  };
 };
 
 export class EditorStore {
+  globalDependencies: Record<string, unknown> = {};
```
[the remark this answers is not in the document]
Comment (MrWindlike) [reply packages/editor/src/services/EditorStore.ts:29]: The `stateManager.dependencies`'s value would be changed when setting the module `properties` as dependencies.

The `globalDependencies`'s usage is to store the origin `dependencies`'s value and we can reset the `dependencies` when exiting the module.

**Comment by MrWindlike** [comment]
> Can we enter array or object as mock data? Will they be converted correctly?

Yes. We would evaluate the `properties` when setting them as `dependencies`.

```js
const evaledDependencies = this.stateManager.deepEval(exampleProperties || {}, {
  fallbackWhenError: () => undefined,
});
```

**Review by tanbowensg — APPROVED** [review]
LGTM

**Pull request merged**